src/components/Activity/activity-types.js

```diff
@@ -101,12 +102,12 @@ const ACTIVITY_TYPES = new Map(
         `,
       }
     },
-    revealVote({ roundId, disputeId }) {
+    heartbeat({ transitions }) {
       return {
-        title: 'Reveal vote',
-        icon: iconAnj,
+        title: 'Heartbeat',
```

Comment:
Shouldn’t this be “Update term”, based on https://github.com/aragon/court-dashboard/pull/227#discussion_r390451950?

Reply:
Yeah totally! Good catch!